src/gpu.cpp

```diff
@@ -74,14 +75,20 @@ nvapi_util();
 void getAmdGpuInfo(){
     int64_t value = 0;
     if (metrics_path.empty()){
-        if (amdgpu.busy) {
-            rewind(amdgpu.busy);
-            fflush(amdgpu.busy);
-            int value = 0;
-            if (fscanf(amdgpu.busy, "%d", &value) != 1)
-                value = 0;
-            gpu_info.load = value;
+#ifdef HAVE_LIBDRM_SAMPLING
+        if (!do_libdrm_sampling) {
```

Comment:
Perhaps it's worth making this global always 0, when `HAVE_LIBDRM_SAMPLING` is not defined? This way one can drop the repeated `ifdef` guards.

Reply:
Actually: it might even better to unconditionally build with it. The library is guaranteed to be present on literally all Linux/FreeBSD/Unixy distro that wants GL or Vulkan.

In addition, as-is it's possible to build w/o that then get hella confused why the toggle does nothing.